src/site/content/en/blog/preload-optional-fonts/index.md

```diff
@@ -0,0 +1,108 @@
+in one of the two ways listed above, until now!
+
+{% Aside %}
+  The [Cumulative Layout Shift](https://web.dev/cls/) metric makes it possible to measure the layout
+  instability on a web page.
+{% endAside %}
+
+## Optional fonts
+
+The `font-display` property uses a timeline of three periods to handle fonts that need to be
+downloaded before they can be rendered:
+
+-  **Block:** Render "invisible" text, but switch to the web font as soon as it finishes loading.
+-  **Swap:** Render text using a fallback system font, but switch to the web font as soon as it
+   finishes loading.
+-  **Fail:** Render text using a fallback system font.
+
+Previously, fonts designated with `font-display: optional` had a 100ms block period and no swap
+period. This means that "invisible" text is displayed very briefly before switching to a fallback
+font. If the font is not downloaded within 100ms, then the fallback font is used and no swapping
+occurs.
+
+<figure class="w-figure">
+  <img src="./previous-behavior-fail.png" alt="Diagram showing previous optional font behavior when font fails to load">
+  <figcaption class="w-figcaption">Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>after</b> the 100ms block period</figcaption>
+</figure>
+
+However, in the case that the font is downloaded before the 100ms block period completes, the custom
+font is rendered and used on the page.
+
+<figure class="w-figure">
+  <img src="./previous-behavior-pass.png" alt="Diagram showing previous optional font behavior when font loads in time">
+  <figcaption class="w-figcaption">Previous <code>font-display: optional</code> behavior in Chrome when font is downloaded <b>before</b> the 100ms block period</figcaption>
+</figure>
+
+Chrome re-renders the page **twice** in both instances, regardless of whether the fallback font
+is used or if the custom font finishes loading in time. This causes a slight flicker of invisible
+text and, in cases when a new font is rendered, a layout jank that moves some of the page's content.
+This occurs easily if the font is stored in the browser's disk cache and can load well before the
+block period is complete (~10ms).
+
+Optimizations have landed In Chrome 82 to entirely remove the first render cycle for optional fonts
+that are preloaded with [`<link rel="preload'>`](https://web.dev/codelab-preload-web-fonts/).
+Instead, rendering is blocked until the custom font has finished loading or a certain period of time
+has passed. This timeout period is currently set at 100ms, but may possibly change in the near
+future to optimize performance.
+
+<figure class="w-figure">
+  <img src="./new-behavior-fail.png" alt="Diagram showing new preloaded optional font behavior when font fails to load">
+  <figcaption class="w-figcaption">New <code>font-display: optional</code> behavior in Chrome when preloaded and font is downloaded <b>after</b> the 100ms block period (no flash of invisible text)</figcaption>
```

Comment:
'behavior in Chrome when **fonts are** preloaded'

I suggest this addition to the caption. As worded, it sounds like it's Chrome that's preloading.

I'm not sure my wording is technically accurate. I do know the current wording needs to change.

Reply:
Definitely technically accurate, and better than the current wording :)

Fixed!